src/index.bundlespec.js

```diff
@@ -0,0 +1,11 @@
+import { i18n, customProperties } from 'ui-kit';
+// import { i18n, customProperties } from '..';
+
+describe('exports', () => {
+  it('should export i18n for three languages', () => {
+    expect(Object.keys(i18n)).toEqual(['en', 'de', 'es']);
```

Comment:
maybe we should check if the shape is correctish? because we have had issues in the past where we go from exporting
```
de: {
   property: value
}
```

to 
```
de: {
 default ...
}
```

Reply:
We can do this in another step. This PR is intended to get the setup up and running. That test was just the simplest one I could come up with.

I'm not too what a test would look like for this. Would it expect that each language doesn't have the  `default` key?

Reply:
Yeah, let's leave it to another PR. I guess the simplest way would just be to check like

expect(i18n.de.UIKit.DateInput.labelRange).toEqual('to');

Reply:
Good idea 👍 I added these now.

The tests would fail in the unlikely case this translation changes, but it's worth ensuring that it works. I also thought about `expect(i18n.en['UIKit.DateInput.labelRange']).toEqual(expect.any(String));` or using a regex, but then I just went for the explicit tests as it is so unlikely to change.